types/react-native-sensor-manager/index.d.ts

```diff
@@ -0,0 +1,44 @@
+// Type definitions for react-native-sensor-manager 0.1
+// Project: https://github.com/kprimice/react-native-sensor-manager
+// Definitions by: Sahin Vardar <https://github.com/SahinVardar>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="react-native" />
+
+export * from 'react-native'
+
+interface SensorManager {
+    startAccelerometer(delay: number): void;
+    stopAccelerometer(): void;
+    startGyroscope(delay: number): void;
+    stopGyroscope(): void;
+    startMagnetometer(delay: number): void;
+    stopMagnetometer(): void;
+    startStepCounter(delay: number): void;
+    stopStepCounter(): void;
+    startThermometer(delay: number): void;
+    stopThermometer(): void;
+    startMotionValue(delay: number): void;
+    stopMotionValue(): void;
+    startOrientation(delay: number): void;
+    stopOrientation(): void;
+    startProximity(delay: number): void;
+    stopProximity(): void;
+    startLightSensor(delay: number): void;
+    stopLightSensor(): void;
+}
+
+declare module 'react-native' {
+    export interface DeviceEventEmitterStatic {
```

Comment:
> I'm not sure if two react-native module declared are conflict.
okay, got it.

>A far better approach would be to define a class named SensorManager, however it use NativeModule directly.

I mean, `react-native-sensor-manager` can be better to do like that.

But never mind, it is okay for typings.